db/migrate/20210823074258_create_reviews.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReviews < ActiveRecord::Migration[6.1]
+  def change
+    create_table :reviews do |t|
+      t.float :rating
+      t.string :comment
+      t.string :source
+      t.references :user, null: false, foreign_key: true
```

Comment:
User NULL should be able to be true since we will let use review anonymously too.